test/state/state.cpp

```diff
@@ -444,7 +444,9 @@ TransactionReceipt transition(const StateView& state_view, const BlockInfo& bloc
 
     if (tx.type == Transaction::Type::blob)
     {
-        const auto blob_fee = tx.blob_gas_used() * block.blob_base_fee;
+        // This uint64 * uint256 cannot overflow, because tx.blob_gas_used has limits enforced
+        // before this stage.
+        const auto blob_fee = tx.blob_gas_used() * block.blob_base_fee.value();
```

Comment:
```suggestion
        const auto blob_fee = tx.blob_gas_used() * *block.blob_base_fee;
```

Reply:
I would add assertion checking that `block.blob_base_fee.ahs_value()`